libs/domain/checkout/src/services/default-checkout.service.ts

```diff
@@ -26,6 +26,7 @@ import {
   PlaceOrderStart,
   PlaceOrderSuccess,
 } from './state';
+import { RouteLinkType } from '@spryker-oryx/router/lit';
```

Comment:
can we get the type from the router entry point? Is there anything lit specific here?